src/background/services/event/__test__/EventEmitter.test.ts

```diff
@@ -0,0 +1,60 @@
+import EventEmitter from "..";
+import { Events, EventHandler, EventName } from "../types";
+
+const events: Events = {
+  identityChanged: () => false,
+  login: () => false,
+  logout: () => false,
+};
+
+jest.mock("nanoevents", (): unknown => ({
+  createNanoEvents: jest.fn(() => ({
+    on: jest.fn().mockImplementation((eventName: EventName, cb: EventHandler) => {
+      events[eventName] = cb;
+    }),
+    emit: jest.fn().mockImplementation((eventName: EventName, payload: number) => {
+      const cb = events[eventName];
+      cb(payload);
+    }),
+  })),
+}));
+
+describe("background/services/event", () => {
+  test("Should be able to add new event listeners", () => {
```

Comment:
All the test titles are started with lowercase and tests must have three sections separated by new lines:
1. Setup: when you initialize what you are going to test, some expected values, spies, mocks
2. Action: actual call of testing function
3. Expect: check return values, mock calls and etc

This comment is related to all the tests in this PR.